ecs-cli/modules/commands/local/local_command.go

```diff
@@ -40,84 +46,151 @@ func LocalCommand() cli.Command {
+				Name:  flagName(flags.TaskDefinitionFile),
+				Usage: flagDescription(flags.TaskDefinitionFile, upCmdName),
 			},
 			cli.StringFlag{
-				Name:  flags.TaskDefinitionFileFlag + ",f",
-				Usage: "Stops and removes all running containers matching the task definition file path",
+				Name:  flagName(flags.TaskDefinitionRemote),
+				Usage: flagDescription(flags.TaskDefinitionRemote, upCmdName),
 			},
 			cli.StringFlag{
-				Name:  flags.TaskDefinitionTaskFlag + ",t",
-				Usage: "Stops and removes all running containers matching the task family or ARN",
+				Name:  flagName(flags.Output),
+				Usage: flagDescription(flags.Output, upCmdName),
 			},
 		},
 	}
 }
 
 func psCommand() cli.Command {
 	return cli.Command{
-		Name:   "ps",
+		Name:   psCmdName,
 		Usage:  "List locally running ECS task containers.",
 		Action: local.Ps,
 		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  flagName(flags.TaskDefinitionCompose),
+				Usage: flagDescription(flags.TaskDefinitionCompose, psCmdName),
+			},
+			cli.StringFlag{
+				Name:  flagName(flags.TaskDefinitionFile),
+				Usage: flagDescription(flags.TaskDefinitionFile, psCmdName),
+			},
+			cli.StringFlag{
+				Name:  flagName(flags.TaskDefinitionRemote),
+				Usage: flagDescription(flags.TaskDefinitionRemote, psCmdName),
+			},
+			cli.BoolFlag{
+				Name:  flagName(flags.All),
+				Usage: flagDescription(flags.All, psCmdName),
+			},
 			cli.BoolFlag{
-				Name:  flags.AllFlag,
-				Usage: "Lists all running local ECS tasks.",
+				Name:  flagName(flags.JSON),
+				Usage: flagDescription(flags.JSON, psCmdName),
+			},
+		},
+	}
+}
+
+func downCommand() cli.Command {
+	return cli.Command{
+		Name:   downCmdName,
+		Usage:  "Stop and remove a running ECS task.",
+		Action: local.Down,
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  flagName(flags.TaskDefinitionCompose),
+				Usage: flagDescription(flags.TaskDefinitionCompose, downCmdName),
 			},
 			cli.StringFlag{
-				Name:  flags.TaskDefinitionFileFlag + ",f",
-				Usage: "Lists all running containers matching the task definition file path",
+				Name:  flagName(flags.TaskDefinitionFile),
+				Usage: flagDescription(flags.TaskDefinitionFile, downCmdName),
 			},
 			cli.StringFlag{
-				Name:  flags.TaskDefinitionTaskFlag + ",t",
-				Usage: "Lists all running containers matching the task family or ARN",
+				Name:  flagName(flags.TaskDefinitionRemote),
+				Usage: flagDescription(flags.TaskDefinitionRemote, downCmdName),
 			},
 			cli.BoolFlag{
-				Name:  flags.JsonFlag,
-				Usage: "Output in JSON format.",
+				Name:  flagName(flags.All),
+				Usage: flagDescription(flags.All, downCmdName),
 			},
 		},
 	}
 }
 
-func createFlags() []cli.Flag {
-	return []cli.Flag{
-		cli.StringFlag{
-			Name:  flags.TaskDefinitionFileFlag + ",f",
-			Usage: "The file name of the task definition to convert",
+func flagName(longName string) string {
+	m := map[string]string{
+		flags.TaskDefinitionCompose: flags.TaskDefinitionCompose + ",c",
```

Comment:
Should we use `fmt.Sprintf` to concatenate strings all the time? Or we can use `+` for short string concatenation.